modules/web/src/app/core/services/auth/service.ts

```diff
@@ -130,10 +130,18 @@ export class Auth {
     const config = this._appConfigService.getConfig();
     if (config.oidc_logout_url) {
       const logoutUrl = new URL(config.oidc_logout_url);
-      if (logoutUrl.searchParams.has('redirectUri')) {
-        logoutUrl.searchParams.set('redirectUri', this._redirectUri);
-      } else {
-        logoutUrl.searchParams.set('redirect_uri', this._redirectUri);
+      switch (config.oidc_provider?.toLowerCase()) {
+        case 'keyclock':
```

Comment:
Please create a constant somewhere for this and don't hard-code it here. Also `s/keyclock/keycloak`.